apps/web/src/components/table/DisplayTable.tsx

```diff
@@ -258,9 +259,9 @@ export default function DisplayTable({
     return (
       <Popover open={filterMenuOpen} onOpenChange={setFilterMenuOpen}>
         <PopoverTrigger asChild>
-          <Button variant="outline" size="sm" className="flex items-center gap-2">
+          <Button variant="outline" size="sm" className="py-4.5 bg-transparent hover:bg-gray-100 dark:hover:bg-gray-500 text-gray-900 dark:text-gray-100 border border-gray-300 dark:border-gray-600 rounded-md shadow-sm flex items-center gap-1">
```

Comment:
```suggestion
          <Button variant="outline" size="sm" className="flex items-center gap-1">
```

same for other places avoid manual styles for buttons